streaming/base/converters/dataframe_to_mds.py

```diff
@@ -82,9 +83,13 @@ def map_spark_dtype(spark_data_type: Any) -> str:
         Raises:
             raise ValueError if no mds datatype is found for input type
         """
-        mds_type = MAPPING_SPARK_TO_MDS.get(type(spark_data_type), None)
+        if issubclass(type(spark_data_type), DecimalType):
+            mds_type = MAPPING_SPARK_TO_MDS.get(DecimalType(), None)
+        else:
+            mds_type = MAPPING_SPARK_TO_MDS.get(spark_data_type, None)
```

Comment:
Hmm, I hate this varname… What do we think about calling this MDS_TYPES? SPARK_TO_MDS? TYPE_MAP? SPARK_TYPE2MDS_TYPE?

Reply:
changed to SPARK_TO_MDS to make it succinct.